### DISCUSSION TITLE
Confused by citation documentation and output

### DISCUSSION THREAD

**Discussion by n8craig** [quarto-dev/quarto-cli#504] [Q&A] [answered]
I am confused by Quarto's treatment of citations and hoping someone can help clarify. The documentation states
> Quarto uses Pandoc to format citations and bibliographies. By default, Pandoc will use the [Chicago Manual of Style](https://chicagomanualofstyle.org/) author-date format, but you can specify a custom formatting using CSL ([Citation Style Language](https://citationstyles.org/)).

However, just above that passage documentation states that 
`Blah Blah [see @knuth1984, pp. 33-35;
also @wickham2015, chap. 1]` 

will produce

> Blah Blah [see Knuth ([1984](https://quarto.org/docs/authoring/footnotes-and-citations.html#ref-knuth1984)), pp. 33-35; also Wickham ([2015](https://quarto.org/docs/authoring/footnotes-and-citations.html#ref-wickham2015)), chap. 1]

This is the output rendered by Quarto. However, to the best of my knowledge this is **_not_** Chicago [author-date](https://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-2.html) format. I'm unaccustomed to seeing Chicago inline citation (or really any author date styles) use `[]` characters when page numbers are referenced. A properly formatted Chicago author-date inline citation would I believe read.

> Blah Blah (see Knuth 1984, pp. 33-35; also Wickham 2015, chap. 1)

I explicitly declared in the YAML `csl: chicago-author-date.csl` but to no avail, the default output was still rendered. I changed it to `evolutionary-anthropology.csl` and got numbered citations as expected, though page numbers still seem wrong to me (but I would need to dig into this further).

Finally, when the equivalent to `Blah Blah [see @knuth1984, pp. 33-35;
also @wickham2015, chap. 1]`  is knitted in an .Rmd outside of Quarto it produces 

> Blah Blah (see Knuth 1984, pp. 33-35; also Wickham 2015, chap. 1)

How to achieve this in Quarto, and perhaps some insight on why it is different would be incredibly helpful. In terms of citation, switching from .Rmd to .Qmd it seems will require some fiddling which feels like a departure from the overall intent of Quarto. I believe I'm missing something.

**Comment by dragonstyle** [comment]
Are you using the visual editor inside of RStudio when creating this citation? It looks to me like in that case, we're writing escape characters before the citation brackets, which converts the citation into a literal set of `[` `]` with `author-in-text` citations inside of it. You can confirm this by switching to source mode and ensuring that you see:

`Blah Blah [see @knuth1984, pp. 33-35; also @wickham2015, chap. 1]`

and not:

`Blah Blah \[see @knuth1984, pp. 33-35; also @wickham2015, chap. 1\]`

If you remove the leading slashes and render, you will see what you expect. 

I'm not sure why we're escaping those braces, but I will take a closer look and fix whatever the issue is. Thanks for letting us know!

**Comment by jjallaire** [comment, marked as answer]
We have these two fixes in the IDE to resolve the problem:

- https://github.com/rstudio/rstudio/commit/184a5be8405d00583f90be39ebec56d389e01688
- https://github.com/rstudio/rstudio/commit/bf397f9881702fb0dcb00a3f5ac2361762d1190f

In the meantime, you can workaround this buggy handling of cite brackets by adding the following `from` field to the document options:

```yaml
---
title: "My Document"
format: html
from: markdown+tex_math_single_backslash
---
```

The addition of `tex_math_single_backslash` effectively turns off the escaping of `[` entirely (as allowing this would conflict with the `\[` syntax for tex display equations implied by `tex_math_single_backslash`

**Comment by n8craig** [comment]
Thank you both very much. Yes, rooting out escape characters solved the issue (I tried to mark both as answers). Switching between source and visual is useful for me so I plan to use the  `markdown+tex_math_single_backslash` option. That is new to me, thank you.

I'm still confused by the documentation and now I'm wondering if an escape character perhaps slipped in elsewhere? ;) The text shown in the documentation isn't Chicago author date, but that is what the documentation indicates it should be, and what I'm accustomed to in Rmd. That had me really confused.

https://quarto.org/docs/authoring/footnotes-and-citations.html#sec-citations

![](https://i.imgur.com/gAVE1iJ.png)

**Comment by jjallaire** [comment]
You are correct! Some extra backslashes did sneak in there (no doubt from the visual editor). This is fixed here: https://github.com/quarto-dev/quarto-web/commit/6cf2b433822e0101ee52472ee3a87015445bd3c8

**Comment by CodesTato** [comment]
Hi everyone; thank you for posting this. I need help with the format of the references/bibliography. I want to use the Vancouver referencing style. It uses numbers (in-text citations appear as numbers), and the references are numbered in the reference list at the end of the document. When I use, for example, the reference @wickham2015 in the Quaero script, it appears as Wickham (2015) in the Quarto output document, and the reference list does not number the reference. How can I make the references appear as numbers in the text and be numbered in the final reference list? Thank you! ;)

**Reply by mcanouil** [reply]
Please open a new thread, since your question is not related to the one from this discussion.
See <https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style> (and <https://quarto.org/docs/visual-editor/technical.html#citations>) to learn how to change reference format/style.
